### PR TITLE
Conv AI SDK

### PR DESCRIPTION
**Pull request by hikmet-demir**

### CODE REVIEW

File: conversational-ai/overview.mdx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+title: "overview"
+description: "Conversational AI: deploy customized, interactive voice agents in minutes."
+---
+asdasdas
```
Comment (jodik) [inline conversational-ai/overview.mdx:5]: Shouldnt we put here some intro as per eg either use js or react library and that mobile is coming soon?

File: conversational-ai/react-sdk.mdx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+##### status
+
+A React state containing the current status of the conversation. 
+
+```js
+const { status } = useConversation();
+console.log(status); // "connected" or "disconnected"
+```
+
+##### mode
```
Comment (jodik) [inline conversational-ai/react-sdk.mdx:119]: can we change this to isSpeaking so it matches the format above (title matches the exported field)

File: conversational-ai/react-sdk.mdx [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+if (!response.ok) {
+    return Response.error();
+}
+
+const body = await response.json();
+const url = body.signed_url; // use this URL for startConversation method.
+```
+
+##### endConversation
```
Comment (jodik) [inline conversational-ai/react-sdk.mdx:94]: Check this file https://github.com/elevenlabs/xi/blob/main/convai-widget/src/sdk/react/index.ts

some methods were renamed eg `endConversation` ->  `endSession`

Also some are missing eg `getOutputByteFrequencyData()`

Also update the packages repo and send Georgy for review

File: conversational-ai/client-sdk.mdx [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+#### Optional callbacks
+
+The options passed to `startSession` can also be used to register optional callbacks:
+
+- **onConnect** - handler called when the conversation websocket connection is established.
+- **onDisconnect** - handler called when the conversation websocket connection is ended.
+- **onMessage** - handler called when a new text message is received. These can be tentative or final transcriptions of user voice, replies produced by LLM. Primarily used for handling conversation transcription.
+- **onError** - handler called when an error is encountered.
+- **onStatusChange** - handler called whenever connection status changes. Can be `connected`, `connecting` and `disconnected` (initial).
+- **onModeChange** - handler called when a status changes, eg. agent switches from `speaking` to `listening`, or the other way around.
```
Comment (jodik) [inline conversational-ai/client-sdk.mdx:104]: Bit out of date check: https://github.com/elevenlabs/xi/blob/main/convai-widget/src/sdk/client/index.ts

E.g. public `getInputByteFrequencyData()` is missing

File: conversational-ai/overview.mdx (hunk 1 above)
Comment (hikmet-demir) [reply conversational-ai/overview.mdx:5]: Overview part is going to come from Sam

File: conversational-ai/react-sdk.mdx (hunk 2 above)
Comment (hikmet-demir) [reply conversational-ai/react-sdk.mdx:119]: done

File: conversational-ai/overview.mdx (hunk 1 above)
Comment (hikmet-demir) [reply conversational-ai/overview.mdx:5]: Let me copy everything from his pr to here

File: conversational-ai/client-sdk.mdx (hunk 4 above)
Comment (hikmet-demir) [reply conversational-ai/client-sdk.mdx:104]: It's not missing 

image

File: conversational-ai/react-sdk.mdx (hunk 3 above)
Comment (hikmet-demir) [reply conversational-ai/react-sdk.mdx]: updated changed the name

It's not missing,
image

**Review by jodik — APPROVED** [review]
Thanks Hikmet

**Pull request merged**